ordered_product/tests/test_ordered_product_page.py

```diff
@@ -0,0 +1,132 @@
+        user.set_password('12345')
+        user.save()
+        client.login(username='testuser', password='12345')
+        response = client.get('/ordered_product/orders')
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/orders.html' in template_names
+        assert set(response.context['orders']) == set(OrderedProduct.filter_user_name(Client(user)))
+        assert response.context['message'] == "No orders found"
+
+    '''
+    @pytest.mark.django_db()
+    def test_nonempty_orders_list(self, client,client0):
+        user = Client.objects.create(user_name=client0.user_name,
+                                     first_name=client0.first_name,
+                                     last_name=client0.last_name,
+                                     password=client0.password,
+                                     area=client0.area)
+        login_check = client.login(username='testuser', password='12345')
+        assert login_check == True
+        response = client.get('/ordered_product/orders')
+        OrderedProduct.order(DeliveryLocation.objects.get(id=int(VALID_DELIVERY_LOCATION_ID)),
+                                                          client,
+                                                          SupplierProduct.objects.get(supplier_product_id=int(VALID_SUPPLIER_PRODUCT_ID)),
+                                                          int(VALID_QUANTITY))
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/orders.html' in template_names
+        assert set(response.context['orders']) == set(OrderedProduct.filter_user_name(str(response.wsgi_request.user)))
+        assert response.context['message'] == ""
+    '''
+
+    @pytest.mark.django_db()
+    def test_change_quantity(self, client):
+        response = client.get('/ordered_product/orders/'+ORDER_PRODUCT_ID, {'quantity': VALID_QUANTITY})
+        assert response.status_code == 302
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert template_names == set()
+
+    @pytest.mark.django_db()
+    def test_delete_order(self, client):
+        response = client.get('/ordered_product/orders/'+ORDER_PRODUCT_ID+',delete')
+        assert response.status_code == 302
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert template_names == set()
+
+    @pytest.mark.django_db()
+    def test_order(self, client):
+        response = client.get('/ordered_product/new_order/')
+        assert response.status_code == 200
+        template_names = set(template.origin.template_name for template in response.templates)
+        assert 'ordered_product/new_order.html' in template_names
```

Comment:
test_order, test_delete_order, and test_change_quantity seems very similar. maybe use `pytest.parametrize`. 
also, are they all just checking that the view exists/returns the right status code? if so, maybe we should change the test name, as `test_delete_order` can suggest we expect to see the functionality of `delete_order` tested.

p.s: if any of those web pages redirects after the functionality is performed, I would like it to be tested.